openzwave-sys/src/notification.rs

```diff
@@ -56,6 +56,52 @@ impl fmt::Display for NotificationCode {
     }
 }
 
+c_like_enum! {
+    ControllerState {
+        ControllerStateNormal = 0,     // No command in progress.
+        ControllerStateStarting = 1,   // The command is starting.
+        ControllerStateCancel = 2,     // The command was cancelled.
+        ControllerStateError = 3,      // Command invocation had error(s) and was aborted.
+        ControllerStateWaiting = 4,    // Controller is waiting for a user action.
+        ControllerStateSleeping = 5,   // Controller command is on a sleep queue wait for device.
+        ControllerStateInProgress = 6, // The controller is communicating with the other device to carry out the command.
+        ControllerStateCompleted = 7,  // The command has completed successfully.
+        ControllerStateFailed = 8,     // The command has failed.
+        ControllerStateNodeOK = 9,     // Used only with ControllerCommand_HasNodeFailed to indicate that the controller thinks the node is OK.
+        ControllerStateNodeFailed = 10 // Used only with ControllerCommand_HasNodeFailed to indicate that the controller thinks the node has failed.
```

Comment:
ah this is interesting, I really thought it was mandatory to prefix the value by the enum name (eg: `ControllerState::Normal`) and that's why I suggested to remove the name from the value because it was redundant (`ControllerState::ControllerStateNormal`).

I missed that we could import the values names like this. I still prefer my suggestion as I think the enum name is a better namespacing mechanism than the value name. (and I think this is still easily replaceable by an automatic search&replace)

But tell me what you think, I wouldn't block on this.